### ISSUE TITLE
Using PyAbel to compute the emission coefficient (W/m3/sr/nm) from a spectral radiance (W/m2/sr/nm)

### ISSUE THREAD

**Issue by enrico1193** [PyAbel/PyAbel#375]
Hi,
in my project I am working with optical emission spectroscopy data of a plasma jet generated in a plasma wind tunnel used to simulate atmospheric entry conditions. What I would like to do is to rebuild the local emission coefficient (W/ m3 /sr/ nm)  starting from the integrated line of sight spectral radiance (W/ m2 / sr / nm) recorded by the spectrometer. Spectral radiance is obtained applying a dedicated intensity calibration.  The standard output of my set-up is an image of 1024 x1024 pixels from an iCCD camera that is spectrally resolved along the horizontal direction and spatially along the vertical one. I have attached an example of a synthetic image. The spectrometer records the plasma jet from the lateral side.

I know that under some assumptions (axial symmetry of the plasma jet and optically thin medium), I can reconstruct the emission coefficient using the Abel inverse transform. I would like to use your package (PyAbel) and in particular the "abel.Transform.trasform" class but I am struggling understanding how to get the correct units using a test case for which I know already the solution

My guess (but I could be wrong) is that the function has no information of the spatial domain so it is performing the integral in a "default" spatial grid which may affect the Abel inversion.  
![radiance](https://github.com/PyAbel/PyAbel/assets/36181258/3d95108e-358d-4404-9967-687330e4052a)

**Comment by DanHickstein** [comment]
Hi @enrico1193! I'm including a drawing of the experiment from your original email message in case others are interested.

Have you successfully applied the abel transform to your data or test case using PyAbel? If you can supply some sample data ([github gist](https://gist.github.com/) can be useful for providing a link) then we can try to help. (A grayscale image could also be a workable start.)

I think that the units should be correct if you utilize the `dr=` parameter for the transform methods. Or, you can just divide your output by `dr`, where `dr` is the pixel size (in meters in your case). @MikhailRyazanov, correct me if I'm wrong here, it's been a while since I've thought about units, Jacobians (and tigers, oh my!). 

![image](https://github.com/PyAbel/PyAbel/assets/1107796/630cf12d-0c02-4915-a6bf-2c841a2f16b3)


https://github.com/PyAbel/PyAbel/issues/234

**Comment by MikhailRyazanov** [comment]
Yes, setting `dr` to the pixel size (not the camera pixel but the corresponding pixel in the object space) in meters should perform the correct scaling. On the other hand, if the magnification is fixed, all this can be simply included into the intensity calibration.

(And, just in case, `abel.Transform.trasform` assumes that the symmetry axis is vertical, so you'll need to transpose your data before and after the transform.)

**Comment by DanHickstein** [comment]
@enrico1193, did you get a chance to try out the `dr=` parameter and see if it works for you?
